tools/Google.Cloud.Tools.ProjectGenerator/Program.cs

```diff
@@ -481,6 +477,9 @@ private static void GenerateCoverageFile(ApiMetadata api, string directory)
             {
                 return;
             }
+            var targetExecutable = new XElement("TargetExecutable", "C:/Program Files/dotnet/dotnet.exe");
+            var targetArguments = new XElement("TargetArguments",
+                $"test --no-build -c Release");
```

Comment:
Unwrap and use a regular string literal for this one?

Reply:
Oh yes, will do.